source/extensions/filters/http/ext_proc/mutation_utils.cc

```diff
@@ -162,7 +162,12 @@ absl::Status MutationUtils::applyHeaderMutations(const HeaderMutation& mutation,
       return absl::InvalidArgumentError("Invalid character in set_headers mutation.");
     }
     const LowerCaseString header_name(sh.header().key());
-    const bool append = PROTOBUF_GET_WRAPPED_OR_DEFAULT(sh, append, false);
+    bool append;
+    if (Runtime::runtimeFeatureEnabled("envoy.reloadable_features.ext_proc_append_default_true")) {
```

Comment:
move this as the default?

Reply:
Keep the runtime flag as false to keep the legacy behavior for a while.

This will buy sometime for ext_proc servers to change to always explicitly encode "append" = "false" or "true", instead of leaving it un-encoded for "false" case.

Reply:
Changed the behavior to have default append value as "true".  This will be a breaking change. 
Keep default append value to false will avoid breaking change now. But, it will be a breaking change when remove that runtime flag.